### PR TITLE
python312Packages.docling-parse: init at 2.0.2

### PR DESCRIPTION
**Pull request by drupol**
This is the first part of an initiative to provide https://github.com/DS4SD/docling to Nix. 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/docling-parse/default.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  dontUseCmakeConfigure = true;
+
+  build-system = [
+    cmake
+    poetry-core
+    pybind11
+    pkg-config
+  ];
+
+  CXXFLAGS = [
```
Comment (natsukium) [inline pkgs/development/python-modules/docling-parse/default.nix:41]: I think `env.NIX_CFLAGS_COMPILE` is better here.

File: pkgs/development/python-modules/mean-average-precision/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  dependencies = [
+    numpy
+    pandas
+  ];
+
+  pythonImportsCheck = [
+    "mean_average_precision"
+  ];
+
```
Comment (natsukium) [inline pkgs/development/python-modules/mean-average-precision/default.nix:34]: Are there no tests?

File: pkgs/development/python-modules/objprint/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  pythonImportsCheck = [
+    "objprint"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  meta = {
+    description = "";
```
Comment (natsukium) [inline pkgs/development/python-modules/objprint/default.nix:34]: The description is missing.

File: pkgs/development/python-modules/mean-average-precision/default.nix (hunk 2 above)
Comment (drupol) [reply pkgs/development/python-modules/mean-average-precision/default.nix:34]: Nope, no tests.

**Comment by drupol** [comment]
@ofborg build loguru

File: pkgs/development/python-modules/docling-core/default.nix [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+  nativeCheckInputs = [
+    jsondiff
+    pytestCheckHook
+    requests
+  ];
+
+  meta = {
+    changelog = "https://github.com/DS4SD/docling-core/blob/${src.rev}/CHANGELOG.md";
+    description = "A python library to define and validate data types in Docling";
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/docling-core/default.nix:54]:
```suggestion
    description = "Python library to define and validate data types in Docling";
```

File: pkgs/development/python-modules/docling-core/default.nix [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    homepage = "https://github.com/DS4SD/docling-core";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ];
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/docling-core/default.nix:57]: Add yourself maybe.

File: pkgs/development/python-modules/docling-parse/default.nix [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+buildPythonPackage rec {
+  pname = "docling-parse";
+  version = "2.0.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "DS4SD";
+    repo = "docling-parse";
+    rev = "v${version}";
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/docling-parse/default.nix:28]:
```suggestion
    rev = "refs/tags/v${version}";
```

**Review by GaetanLepage — CHANGES_REQUESTED** [review]
Not much to say for now. I will run `nixpkgs-review`.

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353183`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/docling-parse/default.nix [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    hash = "sha256-unXGmMp5xyRCqSoFmqcQAZOBzpE0EzgEEBIfZUHhRcQ=";
+  };
+
+  dontUseCmakeConfigure = true;
+
+  build-system = [
+    cmake
+    poetry-core
+    pybind11
+    pkg-config
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/docling-parse/default.nix:38]: cmake and pkg-config should still go to nativeBuildInputs. Not sure about pybind11 but if it is taken from PATH than that one, too.

File: pkgs/development/python-modules/docling-core/default.nix [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  ];
+
+  nativeCheckInputs = [
+    jsondiff
+    pytestCheckHook
+    requests
+  ];
+
+  meta = {
+    changelog = "https://github.com/DS4SD/docling-core/blob/${src.rev}/CHANGELOG.md";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/docling-core/default.nix:53]:
```suggestion
    changelog = "https://github.com/DS4SD/docling-core/blob/v${version}/CHANGELOG.md";
```

File: pkgs/development/python-modules/deepsearch-toolkit/default.nix [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    all = [
+      anyio
+      fastapi
+      uvicorn
+    ];
+    api = [
+      anyio
+      fastapi
+      uvicorn
+    ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/deepsearch-toolkit/default.nix:73]:
```suggestion
  optional-dependencies = rec {
    all = api;
    api = [
      anyio
      fastapi
      uvicorn
    ];
```

File: pkgs/development/python-modules/docling-parse/default.nix (hunk 7 above)
Comment (drupol) [reply pkgs/development/python-modules/docling-parse/default.nix:38]: How can I check `pybind11` ?

File: pkgs/development/python-modules/mean-average-precision/default.nix (hunk 2 above)
Comment (GaetanLepage) [reply pkgs/development/python-modules/mean-average-precision/default.nix:34]: In those cases, we usually explicitly add `doCheck = false;` and a comment that says that there are no tests.
Not a blocker here of course.

**Review by GaetanLepage — APPROVED** [review]
LGTM !

File: pkgs/development/python-modules/mean-average-precision/default.nix (hunk 2 above)
Comment (drupol) [reply pkgs/development/python-modules/mean-average-precision/default.nix:34]: Let's fix it while it's hot.

**Comment by GaetanLepage** [comment]
Formatting mishap apparently.

**Comment by GaetanLepage** [comment]
@ofborg eval

**Comment by drupol** [comment]
@ofborg eval

**Comment by drupol** [comment]
Sorry for all the notifications, I messed up something and `jj` gave me a small headache tonight. Hopefully this is fixed now.

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353183`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/docling-parse/default.nix (hunk 7 above)
Comment (GaetanLepage) [reply pkgs/development/python-modules/docling-parse/default.nix]: Maybe, try to put it in `buildInputs`. If it's not working, then have it in `nativeBuildInputs`.
Anyway, I am pretty sure that it doesn't belong in `build-system`.

Comment (drupol) [reply pkgs/development/python-modules/docling-parse/default.nix]: It works when it is in `buildInputs`. I guess it can stay there?

Comment (GaetanLepage) [reply pkgs/development/python-modules/docling-parse/default.nix]: Yes, I agree.

**Comment by drupol** [comment]
@ofborg eval

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353183`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.docling-parse</li>
    <li>python311Packages.docling-parse.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.docling-parse</li>
    <li>python312Packages.docling-parse.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.docling-parse</li>
    <li>python311Packages.docling-parse.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.docling-parse</li>
    <li>python312Packages.docling-parse.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.docling-parse</li>
    <li>python311Packages.docling-parse.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.docling-parse</li>
    <li>python312Packages.docling-parse.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>loguru</li>
    <li>python311Packages.deepsearch-toolkit</li>
    <li>python311Packages.deepsearch-toolkit.dist</li>
    <li>python311Packages.docling-core</li>
    <li>python311Packages.docling-core.dist</li>
    <li>python311Packages.docling-ibm-models</li>
    <li>python311Packages.docling-ibm-models.dist</li>
    <li>python311Packages.docling-parse</li>
    <li>python311Packages.docling-parse.dist</li>
    <li>python311Packages.marko</li>
    <li>python311Packages.marko.dist</li>
    <li>python311Packages.mean-average-precision</li>
    <li>python311Packages.mean-average-precision.dist</li>
    <li>python311Packages.objprint</li>
    <li>python311Packages.objprint.dist</li>
    <li>python312Packages.deepsearch-toolkit</li>
    <li>python312Packages.deepsearch-toolkit.dist</li>
    <li>python312Packages.docling-core</li>
    <li>python312Packages.docling-core.dist</li>
    <li>python312Packages.docling-ibm-models</li>
    <li>python312Packages.docling-ibm-models.dist</li>
    <li>python312Packages.docling-parse</li>
    <li>python312Packages.docling-parse.dist</li>
    <li>python312Packages.marko</li>
    <li>python312Packages.marko.dist</li>
    <li>python312Packages.mean-average-precision</li>
    <li>python312Packages.mean-average-precision.dist</li>
    <li>python312Packages.objprint</li>
    <li>python312Packages.objprint.dist</li>
  </ul>
</details>

**Pull request merged**